Content.Shared/_RMC14/Xenonids/Neurotoxin/NeurotoxinComponent.cs

```diff
@@ -79,4 +90,65 @@ public sealed partial class NeurotoxinComponent : Component
 
     [DataField, AutoNetworkedField]
     public TimeSpan NextGasInjectionAt;
+
+    [DataField, AutoNetworkedField]
+    public TimeSpan NextNeuroEffectAt;
+
+    [DataField, AutoNetworkedField]
+    public TimeSpan UpdateEvery = TimeSpan.FromSeconds(1);
+
+    [DataField]
+    public Dictionary<string, float> Hallucinations = new()
+    {
+        {"AlienAttack", 0.05f},
+        {"OB", 0.05f},
+        {"Screech", 0.06f},
+        {"CAS", 0.08f},
+        {"Mortar", 0.18f},
+        {"Giggle", 0.27f},
+        {"Sounds", 0.31f}
```

Comment:
It would be better for these strings to be a new enum, both for YAML validation and having data types used represent their use